src/browser-installer/utils.ts

```diff
@@ -129,13 +142,42 @@ const getCacheDir = (envValueOverride = process.env.TESTPLANE_BROWSERS_PATH): st
 export const getRegistryPath = (envValueOverride?: string): string =>
     path.join(getCacheDir(envValueOverride), "registry.json");
 
+export const readRegistry = (registryPath: string): Registry => {
```

Comment:
I decided to extend registry so it would store not only binary paths, but could store anything else
This way we can manage os packages by this registry file too.